api/src/main/java/org/pmiops/workbench/api/ConceptSetsController.java

```diff
@@ -130,9 +134,15 @@ public void setUserProvider(Provider<User> userProvider) {
   private ConceptSet toClientConceptSet(org.pmiops.workbench.db.model.ConceptSet conceptSet) {
     ConceptSet result = TO_CLIENT_CONCEPT_SET.apply(conceptSet);
     if (!conceptSet.getConceptIds().isEmpty()) {
-      result.setConcepts(ImmutableList.copyOf(
-          conceptDao.findAll(conceptSet.getConceptIds()))
-              .stream()
+      Iterable<org.pmiops.workbench.cdr.model.Concept> concepts = conceptDao.findAll(conceptSet.getConceptIds());
```

Comment:
We could (if we wanted to centralize the multimap logic) add a findByIds() method to ConceptService that does this, too.